docker/nginx/snippets/mqtt-ws-upstream-cluster.conf

```diff
@@ -3,7 +3,7 @@
 
 upstream mqtt_ws_cluster {
     least_conn;
-    server mqtt-adapter-1:${MG_MQTT_ADAPTER_WS_PORT};
-    server mqtt-adapter-2:${MG_MQTT_ADAPTER_WS_PORT};
-    server mqtt-adapter-3:${MG_MQTT_ADAPTER_WS_PORT};
+    server mqtt-adapter-1:${SMQ_MQTT_ADAPTER_WS_PORT};
+    server mqtt-adapter-2:${SMQ_MQTT_ADAPTER_WS_PORT};
+    server mqtt-adapter-3:${SMQ_MQTT_ADAPTER_WS_PORT};
```

Comment:
Please add an empty line to the end of the file. Do the same for other `.conf` files.